presto-kafka/src/main/java/io/prestosql/plugin/kafka/schema/confluent/ConfluentModule.java

```diff
@@ -102,6 +107,7 @@ public void configure(Binder binder)
             encoderFactoriesByName.addBinding(AvroRowEncoder.NAME).toInstance((session, dataSchema, columnHandles) -> {
                 throw new PrestoException(NOT_SUPPORTED, "Insert not supported");
             });
+            binder.bind(DispatchingRowEncoderFactory.class).in(SINGLETON);
```

Comment:
Can we bind this one globally ? Same for `DispatchingRowDecoderFactory`

Reply:
Encoder module provides all encoder relates bindings. It is not only for avro, but for all supported in Trino (currently only avro).

Same for Decoder.